test/integration/user/test/interactivity.test.js

```diff
@@ -197,89 +197,114 @@ describe('Interactivity', () => {
                 });
             });
         });
+    });
+
+    describe('when the user move the mouse on the map', () => {
+        describe('and the mouse enters in a feature', () => {
+            it('should fire a featureHover event with a features list containing the entered feature', done => {
+                interactivity.on('featureHover', event => {
+                    expect(event.features[0].id).toEqual(-0);
+                    expect(event.features[0].layerId).toEqual('layer1');
+                    done();
+                });
+                onLoaded(() => {
+                    // Move mouse inside a feature 1
+                    util.simulateMove({ lng: 5, lat: 5 });
```

Comment:
Maybe I'm not seeing something, but this is not testing the new functionality, right? Did we miss those tests before, right?

Reply:
I realized that some tests where wrong placed. The `describe('when the user moves the mouse on the map'` was inside `describe('when the user clicks on the map'`. I simply moved the blocks to the correct place and added the `describe('when the layer changes'` section.